src/ColorContextMenu-Core.package/ColorChooserMorph.class/instance/setHue..st

```diff
@@ -1,4 +1,4 @@
 actions
 setHue: aFraction
 
-	self color: (self color ccmWithHue: aFraction).
\ No newline at end of file
+	self color: (self color ccmWithHue: aFraction)
```

Comment:
Alright, we seem to have different coding styles, but switching back and forth again and again won't solve this for sure 😅

Reply:
was on mistake, reverted